<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingForm</class>
 <widget class="QWidget" name="settingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>442</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>常规</string>
      </attribute>
      <widget class="QLabel" name="lblIDM">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>无</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lblPlayerPot">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>无</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>备用播放器</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtUserName">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnTestPlayerVlc">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>110</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>测试</string>
       </property>
      </widget>
      <widget class="QToolButton" name="btnChoosePlayerVlc">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>37</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>255</y>
         <width>336</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>开关</string>
       </property>
       <widget class="QCheckBox" name="checkClosingWarning">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>关闭窗口警告</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkPlaySound">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>播放音效</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkCheckUpdate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>自动检查更新</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnCheckUpdate">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>80</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>点我手动检查</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBrowserDecodeM3u">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>使用网页解析m3u8</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBrowserDecodeAll">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>50</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>一律使用网页解析</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>167</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>下载器</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>57</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>播放器</string>
       </property>
      </widget>
      <widget class="QToolButton" name="btnChooseIDM">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>167</y>
         <width>37</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>默认接口</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnFinished_1">
       <property name="geometry">
        <rect>
         <x>272</x>
         <y>374</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>完成</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboDefaultInterface">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>220</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>接口1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>接口2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>接口3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>接口4</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="btnTestPlayerPot">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>57</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>测试</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnTestIDM">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>167</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>测试</string>
       </property>
      </widget>
      <widget class="QToolButton" name="btnChoosePlayerPot">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>57</y>
         <width>37</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblPlayerVlc">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>113</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>无</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnChangeBG">
       <property name="geometry">
        <rect>
         <x>266</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>切换背景图</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>其他</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
